procedures/unit-testing-tracing.ipf

```diff
@@ -988,4 +999,88 @@ static Function AnalyzeTracingResult()
 	UTF_Reporting#UTF_PrintStatusMessage(statOut)
 End
 
+Function IUTF_RestoreTracing()
+	variable i, size
+	string path, backupPath, msg
+
+	WAVE/T wvProcs = GetTracedProcedureNames()
+	size = DimSize(wvProcs, UTF_ROW)
+
+	if(size == 1 && !strlen(wvProcs[0]))
+		UTF_Reporting#UTF_PrintStatusMessage("Nothing to restore")
+		return NaN
+	endif
+
+	sprintf msg, "%d procedure files to restore", size
+	UTF_Reporting#UTF_PrintStatusMessage(msg)
+
+	for(i = 0; i < size; i += 1)
+		path = GetProcedurePath(wvProcs[i])
+		backupPath = path + ".backup"
+
+		GetFileFolderInfo/Q/Z=1 backupPath
+		if(V_flag)
+			sprintf msg, "Backup file not found: %s", backupPath
+			UTF_Reporting#UTF_PrintStatusMessage(msg)
+			continue
+		else
+			sprintf msg, "Backup restored for %s", path
```

Comment:
Thats too early here. Should be after `MoveFile`

Reply:
my bad